anvil-python/anvil/commands/haproxy.py

```diff
@@ -98,6 +112,11 @@ def haproxy_questions() -> dict[str, questions.PromptQuestion]:
             default_value="",
             validation_function=validate_key_file,
         ),
+        "tls_mode": questions.PromptQuestion(
+            'TLS termination at HA Proxy ("termination"), passthrough to MAAS ("passthrough"), or no TLS ("")?',
+            default_value="",
+            validation_function=validate_tls_mode,
+        ),
```

Comment:
we should also add a `ssl_ca_cert` in case of pass through for self signed certificates